src/main/java/ua/antibyte/analyzer/dto/CommentDto.java

```diff
@@ -0,0 +1,19 @@
+package ua.antibyte.analyzer.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class CommentDto {
+    private String id;
```

Comment:
```suggestion
    private Long id;
```
I think that the fields that look like Integer/Long values we can parse to Integer/Long fields
this seems not to be a problem with our source file